src/components/Header/Header.tsx

```diff
@@ -31,43 +29,162 @@ export function Header(props: HeaderProps) {
 	const [operatorLogo, setOperatorLogo] = useState(DEFAULT_HEADER.operatorLogo);
 	const [quickAccessItems, setQuickAccessItems] = useState<Array<any>>([]);
```

Comment:
"any" type should be avoided when using typescript. https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
thanks a ticket has been created